src/Components/GameCard.js

```diff
@@ -1,17 +1,19 @@
 import React, { Component } from 'react'
 
 export default class extends Component {
-  
+
   render() {
     const { game, createPopUp } = this.props;
-    
+
     return (
       <div key={game.game} className="carousel-game">
         <div className="game-card" onClick={createPopUp}>
-      
-          <span className="game-card-header">{game.game}</span>
-      
-          <img className="carousel-image" src={game.img} alt="game board"/>
+          <div className="game-card-header-container">
+            <div className="game-card-header-title-container">
+              <span className="game-card-header">{game.game}</span>
+            </div>
+          </div>
+          <img className="game-card-image" src={game.img} alt="game board" />
         </div>
```

Comment:
Additional containers around title are so the words don't move on our hover effects.

Reply:
Nice thinking! That will make for a smoother UI.